src/debugSession/BrightScriptDebugSession.ts

```diff
@@ -158,8 +161,16 @@ export class BrightScriptDebugSession extends BaseDebugSession {
 
             await this.runAutomaticSceneGraphCommands(this.launchConfiguration.autoRunSgDebugCommands);
 
-            util.log(`Exiting any active brightscript debugger`);
-            await this.rokuAdapter.exitActiveBrightscriptDebugger();
+            //delete any previously installed channel
+            try {
+                util.log(`Deleting existing sideloaded channel (if there is one)`);
+                await this.rokuDeploy.deleteInstalledChannel(this.launchConfiguration as RokuDeployOptions);
+            } catch (e) {
+                util.logDebug('Error deleting previously installed channel. Probably not a big deal...', e);
```

Comment:
Don't we get the error messages back now? We could add that to the log. I think it says something like `no installed channel` for example. Could be useful for debugging.

Reply:
I'm including the exception in the logDebug call `, e)` so that should be enough to help with debugging.